packages/features/shell/navigation/Navigation.tsx

```diff
@@ -87,10 +87,10 @@ const getNavigationItems = (orgBranding: OrganizationBranding): NavigationItemTy
     icon: "ellipsis",
   },
   {
-    name: "routing_forms",
-    href: "/apps/routing-forms/forms",
-    icon: "file-text",
-    isCurrent: ({ pathname }) => pathname?.startsWith("/apps/routing-forms/") ?? false,
+    name: "routing",
+    href: "/routing/forms",
+    icon: "split",
+    isCurrent: ({ pathname }) => pathname?.startsWith("/routing/forms") ?? false,
     moreOnMobile: true,
```

Comment:
Was the expected migration supposed to be from `routing-form` to `routing/form`? Why are we doing it as `routing/form`?

Reply:
I think it was said to move away /apps/routing-forms with /routing . So I did this /apps/routing-forms/forms -> /routing/forms.  Pls Correct me if I'm wrong .

Reply:
`migrate /apps/routing-forms/forms to /routing` was said to do.

Reply:
So can we skip /apps/routing-forms to /routing. where it shows the details about Routing Forms ?